addons/editor-dark-mode/addon.json

```diff
@@ -597,6 +597,18 @@
   "injectAsStyleElt": true,
   "versionAdded": "1.0.0",
   "updateUserstylesOnSettingsChange": true,
+  "userscripts": [
+    {
+      "url": "paper.js",
+      "matches": [
+        "https://scratch.mit.edu/projects/*/",
+        "https://scratch.mit.edu/projects/*/editor",
+        "https://scratch.mit.edu/projects/*/fullscreen",
+        "https://scratch.mit.edu/projects/editor"
```

Comment:
can be simplified to "editor"

Reply:
I might make a PR later that changes the matches for all editor addons to "projects".